### PR TITLE
refactor(images): make component extendable - INNO-677

### PR DESCRIPTION
**Pull request by yhuard**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [x] `package.json` is up-to-date and `@ec-europa/ecl-base` is part of the dependencies
- [x] I have checked the dependencies
- [x] I have given the fractal status “ready” to my component
- [x] I have provided tests
- [x] I follow the naming guidelines
- [x] the component supports composition
- [x] there are no hardcoded strings (all content come from the context)
- [x] I have filled the README.md file (at least a few lines)
- [x] My component is listed in the root README
- [x] My PR has the right label(s)

Not relevant:
- [x] ~~I have declared `@define mycomponent` in the SCSS file~~
- [x] ~~I have specified `margin: 0;` on the CSS component~~

### CODE REVIEW

**Comment by yhuard** [comment]
Wouldn't it be better if we provide an interface for the `picture` tag? 
https://www.smashingmagazine.com/2014/05/responsive-images-done-right-guide-picture-srcset/

File: framework/content/ecl-images/ecl-images.twig [hunk 1]
```diff
@@ -1 +1,30 @@
...
+  Accepts:
+    - src (string): specifies the URL of the image
+    - alt (string): specifies an alternate text for the image
+    - extra_classes (string): additional classes to add to the component
+    - extra_attributes (array of { name: '', value: '' })
+#}
+
+{# Internal properties #}
+
+{% set _css_classes = '' %}
```
Comment (kalinchernev) [inline framework/content/ecl-images/ecl-images.twig]: `_css_class` is the version we have at other places.

**Review by kalinchernev — CHANGES_REQUESTED** [review]
About `picture` tag, it looks like a really good idea to provide it as an option!

**Comment by yhuard** [comment]
@kalinchernev INNO-774 as a follow-up for the `picture` tag ;)

**Review by kalinchernev — APPROVED** [review]

**Pull request merged**